webpack.base.js

```diff
@@ -20,10 +20,14 @@ module.exports = ({ entry, mode, filename, types }) => ({
         test: /\.ts$/,
         loader: 'string-replace-loader',
         options: {
-          multiple: [
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏you could probably replace the `string-replace-loader` with the [DefinePlugin](https://webpack.js.org/plugins/define-plugin/), which is part of Webpack, so that would be one dependency less